### DISCUSSION TITLE
batch response: @username can not upload new objects to public fork [repo]

### DISCUSSION THREAD

**Discussion by metavalent** [git-lfs/git-lfs#5341] [Q&A] [answered]
git lfs track "*.mp4"
git add .gitattributes
git add [pathtobigfiles]
git commit -m "add bigfiles"
git push origin gh-pages
Uploading LFS objects:   0% (0/4), 0 B | 0 B/s, done.
batch response: @username can not upload new objects to public fork OrganizationName/branch.github.io
error: failed to push some refs to 'https://github.com/OrganizationName/branch.github.io.git

Any suggestions? Grateful.

**Comment by chrisd8088** [comment, marked as answer]
This looks like you are using GitHub, and specifically may be trying to upload Git LFS objects to a fork of a public repository, which I believe GitHub does not permit as it affects the Git LFS cost billed to the owner of the source repository (not the owner of the fork).  See their [documentation](https://docs.github.com/en/repositories/working-with-files/managing-large-files/collaboration-with-git-large-file-storage#pushing-large-files-to-forks) on this issue, and if you have more questions, I would contact GitHub Support for help, as this project is just for the open source Git LFS client.

**Reply by metavalent** [reply]
Thank you, Chris. For anyone who happens across similar issues, I'm not a super active coder, just the bare minimum to get by. I saw the documentation and was confused because I am the owner of the repository, and now realize that I originally forked a GitHub pages template so I could learn a little bit about Jekyll, and have since moved on from that template, yet the upstream remains. So, the next theory is to detach from that upstream so that I can use LFS on my own repository.

For now, it's off to see the Wizard, that Hallucinating and Lying McLiar Chatty McChatface about how to do that. We're a long, long way from [8088](https://en.wikipedia.org/wiki/Intel_8088) Kansas, eh? Thanks again for the help!
